shell/components/CruResource.vue

```diff
@@ -596,7 +600,7 @@ export default {
                     v-for="(_, slot) of $slots"
```

Comment:
not sure i 100% follow this issue, but if vue3 handles $slots differently can we confirm other places we pass down $slots aren't also affected?

Reply:
This behavior is directly related to Vue3 Slot Unification[^1] and is only an issue in instances where we iterate over slots.

In Vue2, this template would iterate over scoped slots via `v-for="(_, slot) of $scopedSlots"`. Since `$scopedSlots` has been merged with `$slots` in Vue3, we are now inadvertently picking up the default slot along with the rest. 

In previous instances of this error, we filtered for scoped slots via `v-if="typeof $slots[slot] === 'function'". For the offending instances addressed in this PR, `default` was also being treated as a scoped slot and we still don't want to render it in our templates.

I've updated the PR description to call more attention to this behavior. 

[^1]: https://v3-migration.vuejs.org/breaking-changes/slots-unification#slots-unification

Reply:
Gotcha. So we'd be vulnerable to anywhere we previously iterated through $slots that included one that was scoped. I had a look through and we very rarely did it. All good